test/integration/scheduler/framework_test.go

```diff
@@ -216,3 +240,48 @@ func TestPrebindPlugin(t *testing.T) {
 		cleanupPods(cs, t, []*v1.Pod{pod})
 	}
 }
+
+// TestPostbindPlugin tests invocation of postbind plugins.
+func TestPostbindPlugin(t *testing.T) {
+	// Create a plugin registry for testing. Register only a reserve plugin.
+	registry := framework.Registry{prebindPluginName: NewPrebindPlugin, postbindPluginName: NewPostbindPlugin}
+
+	// Create the master and the scheduler with the test plugin set.
+	context := initTestSchedulerWithOptions(t,
+		initTestMaster(t, "prebind-plugin", nil),
+		false, nil, registry, false, time.Second)
+	defer cleanupTest(t, context)
+
+	cs := context.clientSet
+	// Add a few nodes.
+	_, err := createNodes(cs, "test-node", nil, 2)
+	if err != nil {
+		t.Fatalf("Cannot create nodes: %v", err)
+	}
+
+	for i, fail := range []bool{false, true} {
+		pbdPlugin.failPrebind = fail
+		// Create a best effort pod.
+		pod, err := createPausePod(cs,
+			initPausePod(cs, &pausePodConfig{Name: "test-pod", Namespace: context.ns.Name}))
+		if err != nil {
+			t.Errorf("Error while creating a test pod: %v", err)
+		}
+
+		if fail {
+			if err = wait.Poll(10*time.Millisecond, 30*time.Second, podSchedulingError(cs, pod.Namespace, pod.Name)); err != nil {
+				t.Errorf("test #%v: Expected a scheduling error, but didn't get it. error: %v", i, err)
+			}
+		} else {
+			if err = waitForPodToSchedule(cs, pod); err != nil {
+				t.Errorf("test #%v: Expected the pod to be scheduled. error: %v", i, err)
+			}
+		}
+
+		if ptbdPlugin.numPostbindCalled == 0 {
```

Comment:
This does not seem to be correct. Postbind is called when scheduler binds a pod successfully.
This should be moved to `if` and `else` statements above. When pod is unschedulable, postbind plugin should not be called. When pod is schedulable, it should be called.

Reply:
SGTM,thanks